README.md

```diff
@@ -41,10 +41,10 @@ For simulation `qemu_x86` or `qemu_x86_64` board can be used.
 For xen based system `xenvm-qemu` board can be used.
```

Comment:
note: I think that it is needed to update this line too, there isn't board with name `xenvm-qemu`...

Reply:
what about other platforms mentioned above, e.g. `native_posix_64`, `qemu_x86_64` and so on?